package.json

```
@@ -18,7 +18,7 @@
   },
   "license": "BSD-2-Clause",
   "dependencies": {
-    "acorn": "^7.4.0",
+    "acorn": "^8.0.4",
```

```suggestion
    "acorn": "^8.0.5",
```

Can you update this, so we don't forget. 

v8.0.4 doesn't work on some Node versions we support (fixed by https://github.com/acornjs/acorn/commit/0a4167509adee401490464f5d2ff3eeefc2cf341).